spec/models/rent_spec.rb

```diff
@@ -2,8 +2,18 @@
 require 'rails_helper'
 
 RSpec.describe Rent, type: :model do
-  subject(:rent) { build(:rent, user: create(:user, member_at: nil)) }
-  let(:saved) { create(:rent, :good) }
+  let(:member) { create(:user, member_at: Time.zone.now) }
+  let(:n_member) { create(:user, member_at: nil) }
+
+  let(:council) { create(:council) }
+
+  let(:new_rent) { build(:rent, user: member) }
+  let(:new_rent_council) { build(:rent, user: member, council: council) }
+  subject(:rent_n) { build(:rent, user: n_member) }
+
+  let(:rent) { create(:rent, user: member, d_from: Time.zone.now + 15.days,
+                      d_til: Time.zone.now + 15.days + 12.hours) }
```

Comment:
Align the elements of a hash literal if they span more than one line.<br>Expression at 15, 66 should be on its own line.